src/main/java/pl/stormit/tinyurl/service/UrlAnalyticsService.java

```diff
@@ -71,4 +82,22 @@ private Long checkClicksAmountOnShortUrl(UUID urlId) {
             return incMaxClickValue;
         }
     }
+
+    public List<UrlDto> findMostPopularUrls() {
+        PageRequest pageRequest = PageRequest.of(0, 3);
```

Comment:
Move it to some configuration or at least to a constant.

Reply:
Moved to constant